server.js

```diff
@@ -0,0 +1,21 @@
+/*eslint-disable*/
+const express = require("express");
```

Comment:
- [ ] What do you think if you fix your linters instead of disabling them? This is for **[best practices reasons](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md)** and clean code

Reply:
We were instructed to mentain this particular file (server.js) with the linters disabled when we were setting up webpack. Please advise